cdap-spark-core/src/main/scala/co/cask/cdap/app/runtime/spark/SparkMainWrapper.scala

```diff
@@ -214,7 +214,7 @@ object SparkMainWrapper {
           executor.awaitTermination(5L, TimeUnit.SECONDS)
           // Send the complete call
           client.completed(workflowToken)
-          LOG.info("Spark program {} completed. Program RunID: [{}]", programRunId.getProgram, programRunId.getRun)
+          LOG.info("Spark program {} completed. Program RunID: [{}]", programRunId.getProgram, programRunId.getRun:Any)
```

Comment:
@chtyim Using Array not give the desired log. 
"Spark program [PageRankSpark, 0c152701-2b8f-11e7-87b8-42010afa0004] completed. Program RunID: [{}]"
Can you please take another look.